.travis.yml

```diff
@@ -9,6 +9,8 @@ php:
 env:
   - SYMFONY_VERSION="~2.3||~3.0"
 
+sudo: false
```

Comment:
this fixed the build issues composer caused https://github.com/composer/composer/issues/4274#issuecomment-122830215